utils/to-doc.js

```diff
@@ -1,12 +1,16 @@
 'use strict'
 
 var extend = require('pouchdb-extend')
+var now = require('./now')
 
 module.exports = function objectToDoc (object) {
   var doc = extend({}, object, {
     _id: object.id
   })
 
+  doc.updatedAt = now()
+  doc.createdAt = doc.createdAt || doc.updatedAt
```

Comment:
I'm not convinced that this is the place to add the timestamps. I think we should add a dedicated method like "addTimestamps" and call it explicitly. What do you think @nickcolley?